index.bs

```diff
@@ -474,8 +474,15 @@ An implementation may define <dfn export>extension modules</dfn>. These must hav
 [=module name=] that contains a single colon "<code>:</code>" character. The
 part before the colon is the prefix; this is typically the same for all
 [=extension modules=] specific to a given implementation and should be unique for a
-given implementation. Such modules extend the [=local end definition=] and [=remote end definition=]
-providing additional groups as choices for the defined [=commands=] and [=events=].
+given implementation.
+
+Other specifications may define their own WebDriver-BiDi modules that define the protocol.
```

Comment:
Two "define"s in the same sentence.

Reply:
This was your suggestion applied as-is :P 

I suppose the intent here is to replace the second "define" with "extend"

Reply:
Done. Now it reads:

> Other specifications may define their own WebDriver-BiDi modules that extend the protocol.